src/publicatie_tabellen/save_as_csv.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+import shutil
+
+import django.apps
+from django.conf import settings
+from django.db import connection
+from django.utils.module_loading import import_string
+
+logger = logging.getLogger(__name__)
+
+
+class SaveAsCsv:
+    TMP_DIRECTORY = "/tmp/pgdump"
+
+    def main(self, app_names:list):
```

Comment:
This purpose of this method is not clear to me based on the method name. Maybe we don't need this method anymore if we split off the upload_to_blob function to another class. We can handle the removal of the local dump on the calling site (the command handler)

Reply:
moved all to command handler